core/src/main/resources/reference.conf

```diff
@@ -97,7 +97,12 @@ workflow-options {
 
 // Optional call-caching configuration.
 call-caching {
+  # Allows re-use of existing results for jobs you've already run
```

Comment:
I think that's fairly standard in config files, to have commented-out options being set to their default value?